src/test/resources/net/sf/jabref/logic/exporter/ModsExportFormatTestMultipleEntries.bib

```diff
@@ -0,0 +1,20 @@
+% Encoding: UTF-8
+
+@Article{,
+  author   = {Shruthi, Basavaradhya Sahukar and Vinodhkumar, Palani and Selvamani},
+  title    = {Proteomics: A new perspective for cancer.},
+  journal  = {Advanced biomedical research},
```

Comment:
I don't think so. The issue field will not get exported and I couldn't find anywhere that the mods format should  have an issue field.